index.js

```diff
@@ -8,6 +8,9 @@ var zlib = require('zlib');
 var cuid = require('cuid');
 var request = require('request-promise');
 var expressBearerToken = require('express-bearer-token')();
+const backstop = require('backstopjs');
+const path = require('path');
```

Comment:
+1, use of const here was nice because it helped us find our bug the other day. I personally hate the names of the "path" and "url" modules and think they should have been plural because everybody hits this problem.